app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
```

Comment:
**OPTIONAL**
- Please consider using the `after_save` method to call the `update_counter` every time the model changes!

`after_save :update_counter`